nginx/ssl-install.sh

```diff
@@ -130,6 +106,30 @@ select_ssl_certificate_mode(){
 
 }
 
+disable_certbot_compatability(){
+    rm -f /etc/periodic/weekly/reload-nginx.sh || true
+    rm -f /etc/nginx/private/certbot/.well-known/acme-challenge/index.html || true
+}
+
+enable_certbot_compatability(){
+cat > /etc/nginx/private/deploy.sh << EOF
+#!/bin/sh
+cp "\$RENEWED_LINEAGE/fullchain.pem" /etc/nginx/private/cert.pem
+cp "\$RENEWED_LINEAGE/privkey.pem" /etc/nginx/private/key.pem
+EOF
+
+cat > /etc/periodic/weekly/reload-nginx.sh << EOF
```

Comment:
I get the idea. Question: Is it possible to use certbot's ` --deploy-hook` instead?

Reply:
Great! Yeah that makes sense.